components/breadcrumb/navigation.md

```diff
@@ -0,0 +1,40 @@
+---
+title: Navigation
+page_title: Breadcrumb - Navigation
+description: Using the Blazor Breadcrumb for navigating between pages.
+slug: breadcrumb-navigation
+tags: telerik,blazor,breadcrumb,navigation
+published: True
+position: 7
+---
+
+# Breadcrumb for Navigation
+
+The Breadcrumb can be used to navigate between different pages in the application. It can generate the needed links for you through its `UrlField` when [data binding]({%slug breadcrumb-data-binding%}).
+
+In some cases you might prefer to dynamically generate the breadcrumbs based on the current application `Url`. For this scenario you can use the [`NavigationManager`](https://docs.microsoft.com/en-us/dotnet/api/microsoft.aspnetcore.components.navigationmanager?view=aspnetcore-5.0) that the component supports and subscribe to its [`LocationChanged`](https://docs.microsoft.com/en-us/dotnet/api/microsoft.aspnetcore.components.navigationmanager.locationchanged?view=aspnetcore-5.0) event. Thus, you can set new Breadcrumb items on every location change.
```

Comment:
Do we need to hard-code aspnetcore-5.0 in the links? They work without this parameter.

Reply:
Actually we don't, I removed them.